jsonschema/tests/test_cli.py

```diff
@@ -309,17 +310,18 @@ def test_invalid_schema_pretty_output(self):
 
         with self.assertRaises(SchemaError) as e:
             validate(schema=schema, instance="")
-        error = str(e.exception)
+        error = str(e.exception._formatted_message(
```

Comment:
Minor style: jsonschema generally uses this style for continuation:

```suggestion
str(
    e.exception._formatted_message(formatter=cli._json_formatter),
)```